src/components/Sponsors/Sponsors.tsx

```diff
@@ -1,9 +1,33 @@
 import React, { useState, useEffect } from 'react';
 import { useColorMode } from '@docusaurus/theme-common';
 import { sanitize } from 'dompurify';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+
+const cycleInterval = 10000; // 10 seconds
```

Comment:
```suggestion
const cycleIntervalMs = 30000; // 30 seconds
```
Added the suggestion from @sugaarK to give it some time and only refresh every 30 secs.
The "Ms" postfix indicates the unit. Not sure if this is important here. I just wanted to suggest it because in the bf firmware codebase we do that.